pkg/apis/servicecatalog/validation/serviceclass_test.go

```diff
@@ -117,51 +101,6 @@ func TestValidateServiceClass(t *testing.T) {
 			}(),
 			valid: false,
 		},
-		{
-			name: "invalid serviceClass - invalid plan name",
-			serviceClass: func() *servicecatalog.ServiceClass {
-				s := validServiceClass()
-				s.Plans[0].Name = "test-plan.oops"
-				return s
-			}(),
-			valid: false,
-		},
-		{
-			name: "invalid serviceClass - invalid plan guid",
-			serviceClass: func() *servicecatalog.ServiceClass {
-				s := validServiceClass()
-				s.Plans[0].ExternalID = "40d-0983-1b89-★"
-				return s
-			}(),
-			valid: false,
-		},
-		{
-			name: "invalid serviceClass - missing plan guid",
-			serviceClass: func() *servicecatalog.ServiceClass {
-				s := validServiceClass()
-				s.Plans[0].ExternalID = "40d-0983-1b89-★"
```

Comment:
this test case is wrong

Reply:
description is of missing guid, and content is an incorrectly formatted ID.

Reply:
Technically the ID doesn't have to be a GUID, its just recommended. In a follow-on PR we should make sure we're not throwing errors for IDs that are just RECOMMENDED to be GUIDS but in reality can be any non-empty string.

Reply:
the test text saying guid is an artifact of when External ID was a guid.